spinegeneric/cli/generate_figure.py

```diff
@@ -368,6 +374,88 @@ def compute_statistics(df):
     return df, stats
 
 
+def output_text(stats, metric):
+    """
+    # Write statistical results into text file
+    :param stats: dict with stat resutls
+    :param metric: currently processed metric (e.g, dti_fa,...)
+    """
+
+    def check_p_value(p_val):
+        """
+        Check if p-value is lower than 0.01, if so, change it to "<0.01", otherwise, round it to two decimals
+        :param p_val: input p-value
+        :return: p_val: processed p-value (replaced by "<0.01" or rounded to two decimals)
+        """
+        if p_val < 0.01:
+            p_val = "<0.01"
+        else:
+            p_val = round(p_val, 2)
+
+        return p_val
+
+    fname = os.path.join(os.path.abspath(os.curdir), "statistical_results.txt")
+    # Check if file exist, if not create it, if so, append to this file
+    if not os.path.isfile(fname):
+        file = open(fname, "w+")
+    else:
+        file = open(fname, "a+")
+
+    # Write metric name and two blank lines
+    file.write('{}:\n\n'.format(metric))
+    # Find and write highest intra-site COV (rounded up)
+    file.write("The intra-site coefficients of variation (COVs) were averaged for each vendor and found to be all "
+                "just under {}%. ".format(math.ceil(max(stats['cov_intra'].values()) * 100)))
+
+    # Write inter-site COVs and ANOVA p-values
+    file.write("The inter-site COVs (and inter-site ANOVA p-values) were ")
+    for count, vendor in enumerate(stats['cov_inter'].keys()):
+        cov_inter = stats['cov_inter'][vendor] * 100
+        p_val = stats['anova_site'][vendor][1]
+        p_val = check_p_value(p_val)
+        file.write("{:.1f}% (p={}) for {}".format(cov_inter, p_val, vendor))
+        if count == 0:
+            file.write(", ")
+        elif count == 1:
+            file.write(" and ")
+        elif count == 2:
+            file.write(". ")
+
+    p_val_anova = stats['anova_vendor'][1]
+    # Write post-hoc Tukey results if inter-vendor difference was significant
+    if p_val_anova < 0.05:
+        p_val_anova = check_p_value(p_val_anova)
+        file.write("The inter-vendor difference was significant (p={}), with the Tukey test showing significant "
```

Comment:
with that implementation, the output text is `=<0.01` instead of `<0.01`. A suggestion would be to encode the "=" in the `check_p_value()`

Reply:
I am not sure if I get it. When p-val is lower than 0.01 (e.g. 0.009), it is changed to string <0.01. If the p-val is exactly equal to 0.01, p-val falls under _else_ statement and function returns back 0.01.

<img width="237" alt="image" src="https://user-images.githubusercontent.com/39456460/93014558-5a86f800-f5b2-11ea-8f25-73d3b7d3bffa.png">

Reply:
line 428, the “=“ is hardcoded and is always displayed, while it should not if p<0.01

Reply:
Thanks for explanation! Now, I get it how you meant it. Should by fixed in https://github.com/spine-generic/spine-generic/pull/207/commits/01f8108c98b5f57b67f5b1c7f64337e1b83a5bc0